deploy-source/apps/expense-service/src/main/docker/Dockerfile.jvm

```diff
@@ -75,7 +75,7 @@
 #   accessed directly. (example: "foo.example.com,bar.example.com")
 #
 ###
-FROM registry.ocp4.example.com:8443/ubi8/openjdk-17:1.14
+FROM registry.ocp4.example.com:8443/ubi8/openjdk-17:1.18
```

Comment:
Can we update this to Java 21?

Reply:
I guess we can, yes.
But we should? Codestarts use `FROM registry.access.redhat.com/ubi8/openjdk-17:1.18`, that is Java 17.
I don´t see a reason to update, unless there is any benefit.